scripts/server-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# DANGEROUS script (not tested yet) that synchronizes the files
+# in the candidate directory over to the server oceania. It
+# also copies over gmt_data_server.txt.
+
+TOP_DIR=/export/gmtserver/gmt		# The gmt account home dir
+
+CANDIDATE_DIR=${TOP_DIR}/candidate	# Place with updated files
+SERVER_DIR=${TOP_DIR}/data			# Public oceania server dir
+
+# 1. Check we are on the right computer!
+SERVER=$(uname -n)
+if [ ! "${SERVER}" = "gmtserver" ]; then
+	cat <<- EOF  >&2
+	server-release.sh: You must be logged onto the gmtserver, not ${SERVER} to make this target
+	EOF
+	exit 1
+fi
+
+# Here we are logged onto the server and placed ourselves in the candidate server dir
+
+# Sync all data files from candidate to server [UNTESTED]
+echo "rsync -alz --delete ${CANDIDATE_DIR} ${SERVER_DIR}"
```

Comment:
* Are the files (jp2?) compressible? If not, drop the `-z`; it'll only slow down the transfer.
* You do not have trailing slash in the path defined by ``CANDIDATE_DIR`` and ``SERVER_DIR``. That'll copy the *directory* (with all files). Intended?

Reply:
Good point.  Since this is to-from on the same machines there is no network issues and since jp2 cannot really be compressed further and the fewer *.nc files are not big I will remote the z.